Week1/homework/js-exercises/8-type-checker.js

```diff
@@ -0,0 +1,75 @@
+// Exercise 8: Type checker
+
+{
+    'use strict';
+
+    console.log('Exercise 8: Type checker');
+
+    let x = "hello";
+    let y = "world";
+    let book1 = { name: "ABC", year: 2019 };
+    let book2 = { name: "DEF", year: 1909 };
+
+    // check if the data type of one variable is the same as the other
```

Comment:
Logic is correct but I would recommend using `const` instead of `let` and triple equals(===) instead of double equals(==) for comparison in the if block